ursh/indico_ursh/util.py

```diff
@@ -16,19 +16,17 @@
 
 from __future__ import unicode_literals
 
-from werkzeug.exceptions import BadRequest
-
 import requests
 
 
 def request_short_url(original_url):
     from indico_ursh.plugin import UrshPlugin
     api_key = UrshPlugin.settings.get('api_key')
     api_host = UrshPlugin.settings.get('api_host')
-    if not api_key or not api_host:
-        raise BadRequest('API key/host missing - have you set the plugin settings?')
```

Comment:
I would keep raising an error if the values aren't set (after all, someone could send a request to the endpoint manually). Just not a BadRequest. Something as simple as `ServiceUnavailable('Not configured')` should be fine.